utils/oc.py

```diff
@@ -503,12 +514,15 @@ def init_oc_client(self, cluster_info):
                 jump_host = cluster_info.get('jumpHost')
             else:
                 jump_host = None
-            self.set_oc(
-                cluster,
-                OC(server_url, token, jump_host,
-                   settings=self.settings,
-                   init_projects=self.init_projects,
-                   init_api_resources=self.init_api_resources))
+            try:
+                oc_client = OC(server_url, token, jump_host,
+                    settings=self.settings,
+                    init_projects=self.init_projects,
+                    init_api_resources=self.init_api_resources)
+                self.set_oc(cluster, oc_client)
+            except StatusCodeError:
+                logging.info
+                pass
```

Comment:
pushed too early? :)

Reply:
Yes sorry making a change now and testing :disappointed: